homework_10/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,33 @@
+import enums.LoggingLevel;
+
+import java.io.*;
+import java.util.Scanner;
+
+public class FileLoggerConfigurationLoader {
+    public static FileLoggerConfiguration load(String filePath) throws IOException {
+        File configFile = new File(filePath);
+        Scanner scanner = new Scanner(configFile);
+
+        String name = null;
+        LoggingLevel loggingLevel = null;
+        long maxSize = 0;
+        String format = null;
+
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine().trim();
+            if (line.startsWith("FILE:")) {
+                name = line.substring("FILE:".length()).trim();
+            } else if (line.startsWith("LEVEL:")) {
+                String level = line.substring("LEVEL:".length()).trim();
+                loggingLevel = LoggingLevel.valueOf(level.toUpperCase());
+            } else if (line.startsWith("MAX-SIZE:")) {
+                String size = line.substring("MAX-SIZE:".length()).trim();
+                maxSize = Long.parseLong(size);
+            } else if (line.startsWith("FORMAT:")) {
+                format = line.substring("FORMAT:".length()).trim();
+            }
+        }
+        scanner.close();
```

Comment:
Using try-with-resources allows you not to take care about proper opening/closing of the resources.